src/data/item-name.csv

```diff
@@ -1,77 +1,77 @@
-"虹の蹄鉄",""
-"クローバー",""
-"イベントPt",""
-"マニー",""
-"たづなの赤ペン",""
-"フェブラリーS優勝レイ",""
-"高松宮記念優勝レイ",""
-"大阪杯優勝レイ",""
-"桜花賞優勝レイ",""
-"皐月賞優勝レイ",""
-"天皇賞（春）優勝レイ",""
-"ＮＨＫマイルC優勝レイ",""
-"ヴィクトリアマイル優勝レイ",""
-"オークス優勝レイ",""
-"日本ダービー優勝レイ",""
-"安田記念優勝レイ",""
-"宝塚記念優勝レイ",""
-"スプリンターズS優勝レイ",""
-"秋華賞優勝レイ",""
-"菊花賞優勝レイ",""
-"天皇賞（秋）優勝レイ",""
-"マイルCS優勝レイ",""
-"エリザベス女王杯優勝レイ",""
-"ジャパンカップ優勝レイ",""
-"チャンピオンズカップ優勝レイ",""
-"阪神JF優勝レイ",""
-"朝日杯FS優勝レイ",""
-"有馬記念優勝レイ",""
-"ホープフルS優勝レイ",""
-"目覚まし時計",""
-"デイリーレースチケット",""
-"レジェンドレースチケット",""
-"フレンドPt",""
-"ジャパンダートダービー優勝レイ",""
-"JBCスプリント優勝レイ",""
-"JBCクラシック優勝レイ",""
-"JBCレディスクラシック優勝レイ",""
-"帝王賞優勝レイ",""
-"東京大賞典優勝レイ",""
-"はちまき",""
-"ダンベル",""
-"腹筋ローラー",""
-"サポートPt",""
-"サポートカードガチャチケット",""
-"★3確定メイクデビューチケット",""
-"SSR確定メイクデビューチケット",""
-"女神像",""
-"やる気UPスイーツ",""
-"てるてる坊主",""
-"逆さてるてる坊主",""
-"内枠抽選ボール",""
-"外枠抽選ボール",""
-"スタートダッシュ★3引換券",""
-"スタートダッシュSSR引換券",""
-"フォロー枠アップ",""
-"サークルPt",""
-"★2以上確定メイクデビューチケット",""
-"SR以上確定メイクデビューチケット",""
+"短距離シューズ","Short Distance Shoes"
+"マイルシューズ","Mile Shoes"
+"中距離シューズ","Medium Distance Shoes"
+"長距離シューズ","Long Distance Shoes"
+"ダートシューズ","Dirt Shoes"
+"OPグレードメダル","OP Grade Medal"
+"GⅢグレードメダル","G3 Grade Medal"
+"GⅡグレードメダル","G2 Grade Medal"
+"GⅠグレードメダル","G1 Grade Medal"
+"タフネス30","Toughness 30"
+"にんじんゼリーミニ","Carrot Jelly Mini"
+"にんじんゼリー","Carrot Jelly"
+"育成ウマ娘ガチャチケット","Character Gacha Ticket"
+"ジュエル","Jewel"
+"夢の煌めき","Dream Glitter"
+"ルーレットコイン","Roulette Coin"
+"銀の蹄鉄","Silver Horseshoe"
+"金の蹄鉄","Gold Horseshoe"
+"虹の蹄鉄","Rainbow Horseshoe"
+"クローバー","Clover"
+"イベントPt","Event Points"
+"マニー","Money"
+"たづなの赤ペン","Tazuna's Red Pen"
+"フェブラリーS優勝レイ","February S Treasure Scarf"
+"高松宮記念優勝レイ","Takamatsunomiya Kinen Treasure Scarf"
+"大阪杯優勝レイ","Osaka Hai Treasure Scarf"
+"桜花賞優勝レイ","Oka Sho Treasure Scarf"
+"皐月賞優勝レイ","Satsuki Sho Treasure Scarf"
+"天皇賞（春）優勝レイ","Tenno Sho (Spring) Treasure Scarf"
+"ＮＨＫマイルC優勝レイ","NHK Mile C Treasure Scarf"
+"ヴィクトリアマイル優勝レイ","Victoria Mile Treasure Scarf"
+"オークス優勝レイ","Oaks Treasure Scarf"
+"日本ダービー優勝レイ","Japan Derby Treasure Scarf"
+"安田記念優勝レイ","Yasuda Kinen Treasure Scarf"
+"宝塚記念優勝レイ","Takarazuka Kinen Treasure Scarf"
+"スプリンターズS優勝レイ","Sprinters S Treasure Scarf"
+"秋華賞優勝レイ","Shuuka Sho Treasure Scarf"
+"菊花賞優勝レイ","Kikka Sho Treasure Scarf"
+"天皇賞（秋）優勝レイ","Tenno Sho (Fall) Treasure Scarf"
+"マイルCS優勝レイ","Mile CS Treasure Scarf"
+"エリザベス女王杯優勝レイ","Queen Elizabeth Cup Treasure Scarf"
+"ジャパンカップ優勝レイ","Japan Cup Treasure Scarf"
+"チャンピオンズカップ優勝レイ","Champions Cup Treasure Scarf"
+"阪神JF優勝レイ","Hanshin JF Treasure Scarf"
+"朝日杯FS優勝レイ","Asahi Hai FS Treasure Scarf"
+"有馬記念優勝レイ","Arima Kinen Treasure Scarf"
+"ホープフルS優勝レイ","Hopeful S Treasure Scarf"
+"目覚まし時計","Alarm Clock"
+"デイリーレースチケット","Daily Race Ticket"
+"レジェンドレースチケット","Legend Race Ticket"
+"フレンドPt","Friend Points"
+"ジャパンダートダービー優勝レイ","Japan Dirt Derby Trophy Scarf"
+"JBCスプリント優勝レイ","JBC Sprint Trophy Scarf"
+"JBCクラシック優勝レイ","JBC Classic Trophy Scarf"
+"JBCレディスクラシック優勝レイ","JBC Ladies' Classic Trophy Scarf"
+"帝王賞優勝レイ","Teio Sho Trophy Scarf"
+"東京大賞典優勝レイ","Tokyo Daishoten Trophy Scarf"
```

Comment:
I know that this exact term doesn't really exist in English, but I really dislike "Treasure Scarf", it's neither treasure nor a scarf.

First of all, I dunno where "Treasure" came from, not to mention you started using "Trophy" halfway through. It's referring to a "Winner". Treasure or Trophy is redundant here, it's like saying that an athlete won a "Trophy Medal".

About what the item is, it's [this](https://ja.wikipedia.org/wiki/%E5%84%AA%E5%8B%9D%E3%83%AC%E3%82%A4). Japanese wiki says that in English-speaking environment, it's called a "blanket". [This](https://en.wikipedia.org/wiki/Arlington_Million_Trophy) link also calls it a "garland" multiple times (and a blanket once). Even if you translate it literally, it says [lei](https://en.wikipedia.org/wiki/Lei_(garland)), not scarf.

I think "Winner's Blanket" would be the "most" correct term here, but "Winner's Garland" sounds the best (several hits in google for that as well, for example [here](https://www.derbyme.com/2014/01/the-run-for-roses-winners-garland.html)).

Reply:
Wow, really, where did Treasure come from? It's supposed to be Trophy all the way. That's one hell of a brainfart.

I know what the item is IRL, i researched it when I initially translated these and came upon the same results as you, except the garland part. I didn't like calling it a blanket, which is why I went with scarf.

Winner's Garland sounds pretty good to me. I never found that term myself.